src/obfl-specification.html

```diff
@@ -2763,10 +2764,7 @@ <h3 id="L2762">mark-capital-letters</h3>
 
 <p>The overall policy for marking capital letters should be determined by the
 implementing application, for example per job or in a system setting. A user's
-preference should be respected, unless doing so would render the text difficult
-to read or interpret for some reason. The hyphenate attribute can be used when
-detailed control over hyphenation is required, e.g. to ensure proper rendering of
-content that may be difficult to interpret if capital markings are used.</p>
+preference should be respected.</p>
```

Comment:
Saying that there is a "overall policy" isn't really useful unless you also say the default policy can be overwritten, I think.